rxmongo/src/main/scala/akka/contrib/persistence/mongodb/RxMongoReadJournaller.scala

```diff
@@ -123,7 +127,8 @@ class CurrentAllEvents(val driver: RxMongoDriver) extends IterateeActorPublisher
     journal.find(BSONDocument())
       .projection(BSONDocument(EVENTS -> 1))
       .cursor[BSONDocument]()
-      .enumerate()
+      .enumerator()
+      .map(doc => doc) // this is needed for suffix collections
```

Comment:
I cannot really explain this, but running `RxMongoSuffixReadJournalSpec` test without the `map(doc => doc)` line leads to errors like this:
```
[info] - should support the current all events query *** FAILED *** (2 seconds, 888 milliseconds)
[info]   20 was not equal to 0 (ReadJournalSpec.scala:185)
...
[info] - should support the current persistence ids query *** FAILED *** (7 seconds, 157 milliseconds)
[info]   List("1") did not contain all of ("1", "2", "3", "4", "5") (ReadJournalSpec.scala:212)
```
It seems that only the first item of enumerator returned by `driver.getJournalCollections()` is handled, and then nothing happens, the stream seems to be "stopped" or "waiting for something"...
Do you obtain the same errors ?